sdk/search/search-documents/test/utils/setup.ts

```diff
@@ -415,9 +415,11 @@ export async function populateIndex(client: SearchClient<Hotel>): Promise<void>
 
   let count = await client.getDocumentsCount();
   while (count !== testDocuments.length) {
-    await delay(2000);
+    await delay(WAIT_TIME);
     count = await client.getDocumentsCount();
   }
+
+  await delay(WAIT_TIME);
```

Comment:
curious why we need the extra wait here if the count is correct

Reply:
I have seen the scenario in "search get count" (with random index name), the beforeTest method gets executed. The beforeTest method will exit only if the count is 10. 

Now, in the get count test case, I am getting the count again and checking if it is 10. Here it fails. The only reason that I can think of is that the data did not propagate correctly on the server side within the timespan. So, I added this wait. It never failed after adding this.

Reply:
Seems fine, maybe add a comment explaining that?